source/overloading.tex

```diff
@@ -1010,25 +1002,20 @@
 {l|m|m|m}
 \topline
 \hdstyle{Subclause} &   \hdstyle{Expression} &   \hdstyle{As member function} &   \hdstyle{As non-member function} \\ \capsep
-\ref{over.unary}    &   \tcode{@a}   &   \tcode{(a).operator@ (\,)}  &   \tcode{operator@(a)}    \\
-\ref{over.binary}   &   \tcode{a@b}  &   \tcode{(a).operator@ (b)}   &   \tcode{operator@(a, b)} \\
-\ref{over.ass}      &   \tcode{a=b}  &   \tcode{(a).operator= (b)}   &                           \\
-\ref{over.sub}      &   \tcode{a[b]} &   \tcode{(a).operator[](b)}   &                           \\
-\ref{over.ref}      &   \tcode{a->}  &   \tcode{(a).operator->(\,)}  &                           \\
-\ref{over.inc}      &   \tcode{a@}   &   \tcode{(a).operator@ (0)}   &   \tcode{operator@(a, 0)} \\
+\ref{over.unary}    &   \tcode{@a}   &   \tcode{(a).\keyword{operator}@ (\,)}  &   \tcode{\keyword{operator}@(a)}    \\
+\ref{over.binary}   &   \tcode{a@b}  &   \tcode{(a).\keyword{operator}@ (b)}   &   \tcode{\keyword{operator}@(a, b)} \\
+\ref{over.ass}      &   \tcode{a=b}  &   \tcode{(a).\keyword{operator}= (b)}   &                           \\
+\ref{over.sub}      &   \tcode{a[b]} &   \tcode{(a).\keyword{operator}[](b)}   &                           \\
+\ref{over.ref}      &   \tcode{a->}  &   \tcode{(a).\keyword{operator}->(\,)}  &                           \\
+\ref{over.inc}      &   \tcode{a@}   &   \tcode{(a).\keyword{operator}@ (0)}   &   \tcode{\keyword{operator}@(a, 0)} \\
 \end{floattable}
 
 \pnum
-For a unary operator
-\tcode{@}
-with an operand of a type whose cv-unqualified version is
-\tcode{T1},
-and for a binary operator
-\tcode{@}
-with a left operand of a type whose cv-unqualified version is
-\tcode{T1}
-and a right operand of a type whose cv-unqualified version is
-\tcode{T2},
+For a unary operator \tcode{@}
+with an operand of a type whose cv-unqualified version is \tcode{T1},
+and for a binary operator \tcode{@}
+with a left operand of a type whose cv-unqualified version is \tcode{T1}
+and a right operand of a type whose cv-unqualified version is \tcode{T2},
```

Comment:
While we're here: "with an operand of type cv1 T1"?

Reply:
Fixed.